src/cmake/modules/FindOpenColorIO.cmake

```diff
@@ -45,7 +44,18 @@ MACRO(FindOpenColorIO)
         /sw/lib
         /opt/local/lib
         DOC "The OCIO library")
-
+    option(OCIO_STATIC OFF "Use STATIC lib mean use specific config for use")
```

Comment:
I'm sure this is an ESL issue, but I'm not sure I understand the explanation string here?